recipes/apsg/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "apsg" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d2c0b8f68fa3dc68e44d6ee83e5b01d894b1f2567f89fc97ff00749528dcef06
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv"
+  entry_points:
+    - iapsg=apsg.shell:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.14
+    - scipy >=1.0
```

Comment:
in `setup.py` these requirements are not version-constrained, do you need the constraints here? why?

Reply:
I used these constraints originally on Travis for conda package build, but you are right it is likely redundant here. Thanks for the review.